cmd/utils/cmd.go

```diff
@@ -287,6 +291,91 @@ func ExportAppendChain(blockchain *core.BlockChain, fn string, first uint64, las
 	return nil
 }
 
+func ExportHistory(bc *core.BlockChain, dir string, first, last, step uint64) error {
+	log.Info("Exporting blockchain history", "dir", dir)
+	if head := bc.CurrentBlock().Number.Uint64(); head < last {
+		log.Warn("Last block beyond head, setting last = head", "head", head, "last", last)
+		last = head
+	}
+	if err := os.MkdirAll(dir, os.ModePerm); err != nil {
+		return fmt.Errorf("error creating directory: %w", err)
+	}
+	network := "unknown"
+	if name, ok := params.NetworkNames[bc.Config().ChainID.String()]; ok {
+		network = name
+	}
+	var (
+		start    = time.Now()
+		reported = time.Now()
+	)
+	for i := uint64(first); i < last; i += step {
+		gen := func() error {
+			// Open file for Era.
+			fn := path.Join(dir, fmt.Sprintf("%s-%05d.era", network, i/step))
```

Comment:
The nimbus spec proposes the following naming scheme: `<config-name>-<era-number>-<era-count>-<short-historical-root>.era` afaict https://github.com/status-im/nimbus-eth2/blob/stable/docs/e2store.md

Reply:
Ah good point - I've fixed this in the latest. The example shown doesn't have `<era-count>` for some reason. I recall when I download some beacon era files they also didn't so something may be out of sync there. I've omitted it now for that reason.